src/features/controls.ts

```diff
@@ -361,10 +363,14 @@ export const ControlsMixin = (ModelViewerElement:
           this[$promptElement].classList.remove('visible');
         }
 
-        [$onChange]() {
+        [$onChange]({source}: any) {
```

Comment:
Let's define an interface for this event type (probably should define it in the `SmoothControls` module and then import it for use here).